src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -93,13 +94,15 @@ export class ShippingRoute extends Component<
             input: {
               orderId: this.props.order.id,
               fulfillmentType: this.state.shippingOption,
-              shippingName: this.state.name || "",
-              shippingAddressLine1: this.state.addressLine1 || "",
-              shippingAddressLine2: this.state.addressLine2 || "",
-              shippingCity: this.state.city || "",
-              shippingRegion: this.state.region || "",
-              shippingCountry: this.state.country || "",
-              shippingPostalCode: this.state.postalCode || "",
+              shipping: {
+                name: this.state.name || "",
```

Comment:
is it possible to pass `nil` instead of `""` im worried that this will actually set those fields to empty string, i know this was there from before.

Reply:
Yeah, this would be a better semantic improvement - they're nullable

Reply:
Yeah good idea.

Reply:
I've added this, but had to keep country pending a purchase team ticket. There's some form validation that's related to this, too, but we'll get there when we get there 😄